azure-pipelines.yml

```diff
@@ -65,7 +65,7 @@ steps:
 - bash: |
     if [[ $TOXENV == "py" ]]; then
       ./ci_tools/retry.sh python updatezinfo.py
-      python -m tox -- dateutil/test --cov-config=tox.ini --cov=dateutil --junitxml=unittests/TEST-$(Agent.JobName).xml
+      python -m tox -- src/dateutil/test --cov-config=tox.ini --cov=dateutil --junitxml=unittests/TEST-$(Agent.JobName).xml
```

Comment:
I suspect that when `dateutil/test` is `tests/` that specifying the location of the tests won't be necessary anymore, but I may be wrong on that.

Reply:
Indeed. I will happily rework how test discovery works after this PR.

Reply:
Running without the tests folder being passed in will run all tests discovered in the current path. I have when pytest/black/etc try to work on the files of things like venv. I rather leave it with tests/.